docs/howtocite.rst

```diff
@@ -0,0 +1,9 @@
+========================
+How to cite the software
+========================
+
+When using the `ESM-Tools`, please cite the softeware by using the following DOI:
+https://zenodo.org/doi/10.5281/zenodo.3737927. This DOI represents all versions of the software, and will always points to the latest version on zenodo.
+
+To cite a particular version of the software, use one of the following citations that are specific to a software version:
+
```

Comment:
Is there something missing here? I'm not sure we want to maintain a lot of citations here. Probably, we can just ask people to look in Zenodo for the specific version they want to cite?

Reply:
Yes. I though of a list of available versions. But it is probably not practical to maintain it. I give a link to all versions on zenodo.